index.html

```diff
@@ -3,22 +3,22 @@
 <head>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Morz Transport</title>
```

Comment:
mother of god? zasto ovo ispravljas na svakom branchu. procitaj kako funkcionise git merge